spec/integration/dry/rails/container/auto_register/with_block_spec.rb

```diff
@@ -5,8 +5,15 @@
 
   before(:all) do
     Dry::Rails.container do
-      auto_register!("app/workers") do |config|
-        config.memoize = true
+      # auto_register!("app/workers") do |config|
+      #   config.memoize = true
+      # end
+      begin
+        config.component_dirs.add("app/workers") do |dir|
+          dir.memoize = true
+        end
+      rescue
```

Comment:
so, there's something 🐠 going on with reloading, 
or `auto_register!` was a bit smarter when using existing paths?

anyway, with this in place everything is passing, 
so I'm not sure if we should be concerned about this? 
I mean, in real life nobody is really redefining container over and over 🤔